src/analyzer/plugins/analyzerkeyfinder.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+#include <keyfinder/keyfinder.h>
+
+#include <QObject>
+#include <vector>
+
+#include "analyzer/plugins/analyzerplugin.h"
+#include "util/memory.h"
+#include "util/types.h"
+
+namespace mixxx {
+
+class AnalyzerKeyFinder : public AnalyzerKeyPlugin {
+  public:
+    static AnalyzerPluginInfo pluginInfo() {
+        return AnalyzerPluginInfo(
+                "keyfinder:2",
+                QObject::tr("Ibrahim Sha'ath"),
```

Comment:
These strings don't need to be translated

Reply:
good point